specification/data-faas.md

```diff
@@ -0,0 +1,11 @@
+# Semantic conventions for FaaS spans
```

Comment:
Can we say something to the effect of "If Spans following this convention are produced, a Resource of type `faas` MUST exist." (with link to resource conventions)."?

Maybe (I would like that better) put the resource conventions right in this document, instead of data-resource-semantic-conventions, put a link to the faas resource into data-resource-semantic-conventions.

Reply:
I would argue to use the former option. In my opinion, it is better to follow the existing separation where resources are defined in a document and span attributes in another.

Reply:
Well for now it's totally fine, but in future, when the resource conventions document gets too long and we split it up, it would make more sense to mee to have a single `data-faas` file instead of `data-faas` for spans and `data-resource-faas` for resources.